### PR TITLE
Updated code quality range by six.moves.range functions in l10n.py

### PR DESCRIPTION
**Pull request by kdcis**
Replaced range by six.moves.range functions

### CODE REVIEW

**Comment by PeterSurda** [comment]
resolve merge conflicts and check if there's anything left at all in the PR.

**Comment by g1itch** [comment]
> resolve merge conflicts and check if there's anything left at all in the PR.

It's about using the `range()` or `six.moves.range()`. You decide if it's needed here. The code is executed once during the program start.

File: src/l10n.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 import time
 
 from bmconfigparser import BMConfigParser
+from six.moves import range
```
Comment (g1itch) [inline src/l10n.py:10]: isort

File: src/l10n.py [hunk 2]
```diff
@@ -100,7 +101,7 @@ def formatTimestamp(timestamp=None):
     if timestamp is not None and not isinstance(timestamp, int):
         try:
             timestamp = int(timestamp)
-        except:
+        except:  # noqa:E722
```
Comment (PeterSurda) [inline src/l10n.py:104]: `except (ValueError, TypeError)`

**Review by PeterSurda — CHANGES_REQUESTED** [review]

File: src/l10n.py (hunk 1 above)
Comment (kdcis) [reply src/l10n.py:9]: Updated

File: src/l10n.py (hunk 2 above)
Comment (kdcis) [reply src/l10n.py]: Updated

**Review by PeterSurda — APPROVED** [review]

**Review by PeterSurda — APPROVED** [review]

**Pull request merged**